x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/unified_components/query_tab.cy.ts

```diff
@@ -55,7 +57,8 @@ describe(
       cy.get(GET_DISCOVER_DATA_GRID_CELL_HEADER('agent.type')).should('not.exist');
     });
 
-    context('flyout', () => {
+    // these tests are skipped until we implement the expandable flyout in the unified table for timeline
+    context.skip('flyout', () => {
```

Comment:
@michaelolo24 let me know if you're ok skipping these tests. Explanation of why I think we should do this [here](https://github.com/elastic/kibana/issues/180679#issuecomment-2073279559)

Reply:
Yea, that's fine. I think the flyout in timeline being enabled will align with when the unified table is ready so no worries, we can move skip this. Fyi @logeekal